### PR TITLE
Update README to clarify tkinter install

### PR DESCRIPTION
**Pull request by berrak**
Update README with installation tips for tkinter module on Linux based platforms. Various issue shows up from time to time, e.g.mentioned in issue#5.

### CODE REVIEW

**Comment by JamesH65** [comment]
I just ran the generator with gui on a Pi without any installation of tkinter and it worked OK, so I assume its already installed? I cannot comment on Debian Ubuntu.

**Comment by berrak** [comment]
The multitude of RPI OS options probably requires a bit more testing with various variants on RPI.
RPI has "Raspberry Pi OS with desktop and recommended software" and the  "Raspberry Pi OS with desktop", and without the extra software and "Debian Buster with Raspberry Pi Desktop". But if the intention is to use RPI for C/C++ development, I guess for 64-bit IDE compatibility reasons, the preferred option would be Raspios_arm64 from here: https://downloads.raspberrypi.org/raspios_arm64/images/
Which RPI OS works for you? I use Debian/testing (Bullseye .. to be released shortly) -- I had to install python3-tk.
For the latest Ubuntu version, it must be confirmed, to be sure.

**Comment by JamesH65** [comment]
Odd, I'm using Rasberry Pi OS (Buster), and that seemed to have all I needed to run the GUI. Will heck with the distro maintainer to see if it is no longer packaged on Bullseye..

**Comment by berrak** [comment]
I fired up virtualbox and installed the latest release of Ubuntu, i.e 2021-4. and confirmed that the `package for `tkinter was not installed.
Thus Debian+Ubuntu does not have it installed by default. I have one RPI3, but I do not want to reinstall Rasp-OSx64, on the existing SD card. I will find another SD-card and do the test and come back.
root@berrak-virtualbox: apt-cache policy python3-tk
python3-tk:
  Installed: (none)
  Candidate: 3.9.5-0ubuntu3~21.04

**Comment by JamesH65** [comment]
I'm generally happy with the PR, I just want to make sure its accurate!

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ be added to the project using the `--feature` options, this can be used multiple
 
 ## GUI version
 
-The GUI version of the tool, run by adding `--gui` to the command line, uses `tkinter` to provide a platform agnostic script that will run on Linux, Mac and Windows. All the options from the command line tool are also supported in the GUI.
+The GUI version of the tool, run by adding `--gui` to the command line, uses `tkinter` to provide a platform agnostic script that will run on Linux, Mac and Windows. All the options from the command line tool are also supported in the GUI. Install `python3-tk` package for GUI support on Ubuntu/Debian/RPI platforms.
```
Comment (lurch) [inline README.md:50]: Perhaps "It may be necessary to install the ..." rather than "Install ..." ?  (Based on James' comments)

Comment (berrak) [reply README.md:50]: Absolutely, I will make the correction and do the final test on the RPI3 I have.

**Comment by berrak** [comment]
Verified that RPI image 2021-05-07-raspios-buster-arm64.img includes python3-tk, thus updated clarification, and leaving only Ubuntu and Debian.

**Review by lurch — APPROVED** [review]
LGTM

**Comment by JamesH65** [comment]
Thanks!

**Pull request merged**